extensions/wikia/CommunityHeader/templates/CommunityHeader_localNavigation.php

```diff
@@ -73,13 +73,38 @@ class="wds-dropdown-level-2__toggle"
 				</div>
 				<div class="wds-is-not-scrollable wds-dropdown__content">
 					<ul class="wds-list wds-is-linked wds-has-bolded-items">
-						<? foreach ( $navigation->exploreItems as $exploreItem ): ?>
-							<li>
-								<a href="<?= $exploreItem->href ?>"
-									<? if ( $isPreview ): ?>target="_blank"<? endif; ?>
-									data-tracking="<?= $exploreItem->tracking ?>"
-								><?= $exploreItem->label->renderInContentLang() ?></a>
-							</li>
+						<? foreach (  $navigation->exploreItems as $index => $exploreItem ): ?>
```

Comment:
Can we either refactor that or add some comments? It's very un-readable right now.

Reply:
Considering I borrowed all of the logic from the code above, I tend to agree. A lot of knot untangling to get this to work.

Reply:
Just the difference between `$exploreItem->items` and the one without it. It doesn't seem to weird, just complex.

Reply:
Added a few comments that should help a little.